llama-index-integrations/llms/llama-index-llms-premai/llama_index/llms/premai/base.py

```diff
@@ -116,18 +80,12 @@ def __init__(
         self,
         project_id: int,
         premai_api_key: Optional[str] = None,
-        session_id: Optional[int] = None,
         model: Optional[str] = None,
         system_prompt: Optional[str] = None,
         max_tokens: Optional[str] = 128,
         temperature: Optional[float] = 0.1,
-        top_p: Optional[float] = 0.7,
         max_retries: Optional[int] = 1,
-        tools: Optional[Dict[str, Any]] = None,
-        frequency_penalty: Optional[float] = None,
-        presence_penalty: Optional[float] = None,
-        logit_bias: Optional[dict] = None,
-        seed: Optional[int] = None,
```

Comment:
thats a lot of removed argument, leading this to be a breaking change right now.

Should we add a **kwargs option to the constructor, and if any of these old args are passed in, we give an appropriate message to the user?

Reply:
Hi, thanks for pointing this out, I added an another commit to handle that, let me know if that works.